entities/global.ent

```diff
@@ -606,5 +606,653 @@
+<!ENTITY url.xattr "http://savannah.nongnu.org/projects/attr/">
+<!ENTITY url.xchm "http://xchm.sourceforge.net/">
+<!ENTITY url.xdebug "http://xdebug.org/">
+<!ENTITY url.xdiff "http://www.xmailserver.org/xdiff-lib.html">
+<!ENTITY url.xhprof.docs.facebook "http://web.archive.org/web/20110514095512/http://mirror.facebook.net/facebook/xhprof/doc.html">
+<!ENTITY url.xhprof.gui.fork1 "http://github.com/preinheimer/xhprof">
+<!ENTITY url.xinclude.spec "http://www.w3.org/TR/xinclude/">
+<!ENTITY url.xitami "http://www.xitami.com/">
+<!ENTITY url.xforms "http://www.w3.org/MarkUp/Forms/">
+<!ENTITY url.xforms.htmlauthors "http://www.w3.org/MarkUp/Forms/2003/xforms-for-html-authors.html">
+<!ENTITY url.xkcd "http://xkcd.com/">
+<!ENTITY url.xml "http://www.w3.org/XML/">
+<!ENTITY url.xmlrpc "http://www.xmlrpc.com/">
+<!ENTITY url.xmlrpc-epi "http://xmlrpc-epi.sourceforge.net/">
+<!ENTITY url.xslt "http://www.w3.org/TR/xslt">
+<!ENTITY url.yaf.benchmarks "http://www.laruence.com/2011/12/02/2333.html">
+<!ENTITY url.yaml "http://www.yaml.org/">
+<!ENTITY url.yaml.libyaml "http://pyyaml.org/wiki/LibYAML">
+<!ENTITY url.yaz "http://www.indexdata.dk/yaz/">
+<!ENTITY url.yaz.zoom.ext "http://www.indexdata.dk/yaz/doc/zoom.tkl">
+<!ENTITY url.yaz.ftp.win32 "http://ftp.indexdata.dk/pub/yaz/win32/">
+<!ENTITY url.yaz.ftp "http://ftp.indexdata.dk/pub/yaz/">
+<!ENTITY url.yaz-ccl "http://www.indexdata.dk/yaz/doc/tools.tkl#CCL">
+<!ENTITY url.yaz-phpyaz "http://www.indexdata.dk/phpyaz/">
+<!ENTITY url.yaz.ill "http://www.collectionscanada.ca/iso/ill/stanprf.htm">
+<!ENTITY url.yaz-proxy "http://www.indexdata.dk/yazproxy/">
+<!ENTITY url.yaz.specs "http://www.collectionscanada.ca/iso/ill/document/standard/z-ill-1a.pdf">
+<!ENTITY url.yaz-loc "http://www.loc.gov/z3950/agency/">
+<!ENTITY url.yaz-loc-bib1 "http://www.loc.gov/z3950/agency/defns/bib1.html">
+<!ENTITY url.yaz-marcxml "http://www.loc.gov/standards/marcxml/">
+<!ENTITY url.yaz.z3950.4 "http://www.loc.gov/z3950/agency/markup/04.html#3.2.1.1.4">
+<!ENTITY url.zend "http://www.zend.com/">
+<!ENTITY url.zend.ide "http://www.zend.com/en/products/studio/">
+<!ENTITY url.zeromq "http://zeromq.org/">
+<!ENTITY url.zlib "http://www.zlib.net/">
+<!ENTITY url.zlib.manual "http://www.zlib.net/manual.html">
+<!ENTITY url.zookeeper "https://zookeeper.apache.org/">
+<!ENTITY url.zookeeper.doc.cbinding "https://zookeeper.apache.org/doc/trunk/zookeeperProgrammers.html#C+Binding">
+
+<!ENTITY spec.http1.1  'http://tools.ietf.org/html/rfc7231'>
+<!ENTITY spec.soap1.1  'http://www.w3.org/TR/soap11/'>
+<!ENTITY spec.soap1.2  'http://www.w3.org/TR/soap12/'>
+<!ENTITY spec.wsdl     'http://www.w3.org/TR/wsdl'>
+
+<!ENTITY book.applied.cryptography 'Applied Cryptography by Schneier (ISBN 0-471-11709-9)'>
+<!ENTITY book.programming.classics "Programming Classics: Implementing the World's Best Algorithms by Oliver (ISBN 0-131-00413-1)">
+
+<!ENTITY email.harald                'harald.radi@nme.at'>
```

Comment:
I'm not sure this is a good idea?

Reply:
This was already part of the global entities file, I just accidentally duplicated all but the acronyms part of it.

Reply:
@Girgias, heh

https://github.com/php/doc-en/blob/72605fc9de80bd30f50b112e447326dfe6c2a523/faq/com.xml#L227-L228

Reply:
I wonder if they would even reply :')